README.md

```diff
@@ -38,27 +42,37 @@ for hands-on MLOps tutorials using CML!
 
 ## Table of contents
 
-1. [Usage](#usage)
-2. [Getting started (tutorial)](#getting-started)
-3. [Using CML with DVC](#using-cml-with-dvc)
-4. [Using self-hosted runners](#using-self-hosted-runners)
-5. [Install CML as a package](#install-cml-as-a-package)
-6. [Example Projects](#see-also)
+1. [Setup (GitLab, GitHub, BitBucket)](#setup)
+2. [Usage](#usage)
+3. [Getting started (tutorial)](#getting-started)
+4. [Using CML with DVC](#using-cml-with-dvc)
+5. [Advanced Setup (Self-hosted, local package)](#advanced-setup)
+6. [Example projects](#see-also)
 
-## Usage
+## Setup
 
 You'll need a GitHub or GitLab account to begin. Users may wish to familiarize
 themselves with [Github Actions](https://help.github.com/en/actions) or
 [GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).
 Here, will discuss the GitHub use case.
 
-- **GitLab users**: Please see our
-  [docs about configuring CML with GitLab](https://github.com/iterative/cml/wiki/CML-with-GitLab).
-- **Bitbucket Cloud users**: Please see our
-  [docs on CML with Bitbucket Cloud](https://github.com/iterative/cml/wiki/CML-with-Bitbucket-Cloud).
-  _Bitbucket Server support estimated to arrive by May 2021._
-- **GitHub Actions users**: The key file in any CML project is
-  `.github/workflows/cml.yaml`:
+### GitLab
+
+Please see our docs on
+[CML with GitLab](https://github.com/iterative/cml/wiki/CML-with-GitLab) and in
+particular the
+[personal access token](https://github.com/iterative/cml/wiki/CML-with-GitLab#variables)
+requirement.
+
+### Bitbucket Cloud
+
+Please see our docs on
+[CML with Bitbucket Cloud](https://github.com/iterative/cml/wiki/CML-with-Bitbucket-Cloud).
+_Bitbucket Server support estimated to arrive by mid 2021._
+
+### GitHub Actions
+
+The key file in any CML project is `.github/workflows/cml.yaml`:
```

Comment:
GitHub Actions' GitLab counterpart would be GitLab CI/CD, not GitLab as a whole. Should we standardize on either CI/CD solution names or platform names?

Reply:
Technically since we're targeting both the CI/CD as well as e.g. PR comments, "as a whole" might be better (so GitLab/BitBucket/GitHub)? On the downside misses out on possible matches with searches for `GitHub Actions`/`GitLab CI/CD/`/`BitBucket Pipelines`.

PS is it BitBucket or Bitbucket? And Cloud or Server or Pipelines?

Reply:
>_P.S. is it BitBucket or Bitbucket? And Cloud or Server or Pipelines?_

<details><summary>It's Bitbucket, not BitBucket</summary>

![Captura de pantalla 2021-06-28 a las 22 55 00](https://user-images.githubusercontent.com/11387611/123702939-09f03b00-d864-11eb-9e6e-38742cac0538.png)
![Captura de pantalla 2021-06-28 a las 22 55 42](https://user-images.githubusercontent.com/11387611/123702943-0bb9fe80-d864-11eb-9c1d-0550e062f2ec.png)

</details>

...and we don't support Bitbucket Pipelines yet. That would be the equivalent of GitHub Actions and GitLab CI/CD.

Reply:
> _Technically since we're targeting both the CI/CD as well as e.g. PR comments, "as a whole" might be better (so GitLab/BitBucket/GitHub)?_

Agreed!

Reply:
> _On the downside that would miss out on possible matches with searches for GitHub Actions/GitLab CI/CD//BitBucket Pipelines._

We have lots of opportunities to use those in the documentation that is specific to the `cml-runner` command. Would it be enough for search engine optimization?

Reply:
> _`GitHub Actions`/`GitLab CI/CD/`/`BitBucket Pipelines`_

Agh! My eyes! 😄 

<details><summary>Extra slash</summary>

![(](https://user-images.githubusercontent.com/11387611/123703845-3ce6fe80-d865-11eb-9170-e6c1928382b6.png)
</details>


<details><summary>Slash injection</summary>

![ted_talk](https://user-images.githubusercontent.com/11387611/123703888-48d2c080-d865-11eb-89e1-1219c7c1c6ab.png)
</details>

<details><summary>Repeated word illusion</summary>

![9781660874804_p0_v2_s1200x630](https://user-images.githubusercontent.com/11387611/123703674-fee9da80-d864-11eb-9c01-b7fb32c9571f.jpg)
</details>

*** 

Solutions like GitHub Actions/GitLab CI/CD/BitBucket Pipelines... what is CD, by the way? 😈  Whoever named it had the [Bobby Tables](https://xkcd.com/327/) Syndrome.